vagrant_repo/data_bags/cerner_splunk/license-vagrant.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "license-vagrant",
+  "auto_generated_pool_size": "100KiB", 
+  "pools":{
+    "pool1": {
+      "max_size": "200KiB",
+      "GUIDS": "aae84706-a70d-4134-951d-93a6541011fa, 9d8f6bb8-49b2-463e-afdd-aedf267526e4"
```

Comment:
Updated in this commit: 3281400a236b3e8d6304d4c130fbf7edfc3db53e

I also noticed that I was reading the license quota from the license file even before the license file was created by the license_server.rb recipe. Hence I have also included a fix for that in this commit.